core/include/drivers/gpio.h

```diff
@@ -6,6 +6,32 @@
 #ifndef __GPIO_H__
 #define __GPIO_H__
 
+#include <dt-bindings/gpio/gpio.h>
+#include <kernel/dt.h>
```

Comment:
Once https://github.com/OP-TEE/optee_os/pull/5993 is merged, you should drop `#include <kernel/dt.h>`.

Reply:
Done